requirements/doc-requirements.txt

```diff
@@ -16,3 +16,4 @@ torchvision>=0.12.0
 lightning>=1.8.1
 scrapy
 ipywidgets>=8.1.1
+incremental<24.7.0
```

Comment:
```suggestion
# incremental==24.7.0 requires setuptools>=61.0, which causes https://github.com/mlflow/mlflow/issues/8635
incremental<24.7.0
```

Reply:
https://github.com/pypa/setuptools/issues/2993 seems related.

Reply:
I tried python 3.9 (3.8 is used currently) but it failed https://app.circleci.com/pipelines/github/mlflow/mlflow/39417/workflows/77c58eb0-6f88-448f-b0a1-478497709c92/jobs/116794